cmd/tenant/main.go

```diff
@@ -149,6 +179,218 @@ func usage(c *cli.Context) (err error) {
 // Client Commands
 //===========================================================================
 
+type oldMember struct {
+	OrgID        ulid.ULID       `msgpack:"org_id"`
+	ID           ulid.ULID       `msgpack:"id"`
+	Email        string          `msgpack:"email"`
+	Name         string          `msgpack:"name"`
+	Role         string          `msgpack:"role"`
+	Status       db.MemberStatus `msgpack:"status"`
+	Created      time.Time       `msgpack:"created"`
+	Modified     time.Time       `msgpack:"modified"`
+	DateAdded    time.Time       `msgpack:"date_added"`
+	LastActivity time.Time       `msgpack:"last_activity"`
+}
+
+func migrateProfiles(c *cli.Context) (err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+
+	dry := c.Bool("dry-run")
+
+	// Open the CSV file containing the list of organizations
+	var orgsFile *os.File
+	if orgsFile, err = os.Open(c.String("orgs")); err != nil {
+		return cli.Exit(err, 1)
+	}
+	defer orgsFile.Close()
+
+	// Open the file to write the report to
+	var reportFile *os.File
+	report := c.String("report")
+	if report != "" {
+		if reportFile, err = os.Create(c.String("report")); err != nil {
+			return cli.Exit(err, 1)
+		}
+		defer reportFile.Close()
+	}
+
+	// Read the header
+	var header []string
+	reader := csv.NewReader(orgsFile)
+	if header, err = reader.Read(); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	// Ensure header fields are correct
+	if header[0] != "ID" || header[1] != "Name" || header[2] != "Domain" {
```

Comment:
Unless the CSV already export exists, I would usually name the headers the same name as the columns in the database, which are lower snake case. 

```suggestion
	if header[0] != "id" || header[1] != "name" || header[2] != "domain" {
```

Reply:
I already have the CSV export from running SC-21250, this matches the header format defined there.